client/src/js/app.js

```diff
@@ -746,11 +746,17 @@
     });
 
     $translateProvider.useSanitizeValueStrategy('escape');
-
-    //TODO Try and assign the previous sessions language key here
+  
     $translateProvider.preferredLanguage('fr');
   }
 
+  function localeConfig(tmhDynamicLocaleProvider) { 
+
+    // TODO Hardcoded default translation/ localisation
+    tmhDynamicLocaleProvider.localeLocationPattern('/i18n/locale/angular-locale_{{locale}}.js');
+    tmhDynamicLocaleProvider.defaultLocale('fr-fr');
```

Comment:
This is minor, but it might be best to bundle the [Belgian French](https://github.com/angular/angular.js/blob/master/src/ngLocale/angular-locale_fr-be.js) locale as default, rather than the France French.